src/app/(homepage)/SignUp.js

```diff
@@ -9,52 +9,65 @@ import Typography from "@mui/material/Typography";
 import { useTheme } from "@mui/material/styles";
 
 
-function SignUpForm() {
-  const theme = useTheme();
-
+function SignUpField() {
```

Comment:
What is the logic behind dividing into such modular components?
I suppose a whole SignUpForm to be reused, but not AgreementSection, SignUpForm